docs/csharp/language-reference/operators/toc.yml

```diff
@@ -4,6 +4,9 @@
   - name: Arithmetic operators
     href: arithmetic-operators.md
     displayName: ++, --, +, -, *, /, %
+  - name: Boolean logical operators
```

Comment:
Not related to this PR, but the H2 anchors in the TOC are not officially supported. We should remove them.

(Yes, they do work now, but that's not guaranteed behavior).

Reply:
@BillWagner does that mean that the following:
```
- name: / Operator
  href: arithmetic-operators.md#division-operator-
```

should be replaced with:
```
- name: / Operator
  href: arithmetic-operators.md
```

??

Reply:
Sorry, I should have been more clear.

I want to try removing those items from the TOC. Ideally, readers find the individual operators as needed. If not, we'll address (and I'll likely keep pushing for a better experience for H2s in the TOC.

Reply:
> I want to try removing those items from the TOC.

Ah! OK, let's do that.

By the way, with the new TOC feature that filters by `displayName`, maybe it's time to revise the UX. The current one:
![image](https://user-images.githubusercontent.com/15279990/55640793-9bc78580-57cc-11e9-9802-fdf957738769.png)


It says "Filter by title". If most of the individual operator entries are removed and it's obvious that there are no titles in TOC with "+" in it, I wouldn't even try to type "+" into this field.

Reply:
I asked for a change See MicrosoftDocs/feedback#1359